spec/configuration_spec.rb

```diff
@@ -202,6 +211,40 @@ def call(status)
         connecttimeout: 0.2,
       ))
     end
+
+    it 'does not override per-request timeout' do
+      client = NxtHttpClient::Client.make do
+        configure do |config|
+          config.base_url = 'httpstat.us?sleep=1000'
+          config.timeout_seconds(total: 0.5, connect: 0.2)
+        end
+      end
+
+      expect(client.build_request('', timeout: 10).options).to match(hash_including(
+        timeout: 10,
+        connecttimeout: 0.2,
+      ))
```

Comment:
nitpick: 

```suggestion
      expect(client.build_request('', timeout: 10).options).to include(
        timeout: 10,
        connecttimeout: 0.2,
      )
```

There are a couple other places where match + hash_including can be replaced with include.

Reply:
Will do. include, contains_exactly, match, match_array - I _always_ mix up these RSpec matchers 😅